workflowproj/operator.go

```diff
@@ -34,6 +34,8 @@ const (
 	ApplicationPropertiesFileName = "application.properties"
 	// LabelApp key to use among object selectors, "app" is used among k8s applications to group objects in some UI consoles
 	LabelApp = "app"
+	// LabelService key to use among object selectors
+	LabelService = "sonataflow.org/service"
```

Comment:
@masayag can you please fix this in a follow up PR?

```go
LabelService = metadata.Domain + "/service"
```

Reply:
@ricardozanini done here: https://github.com/apache/incubator-kie-kogito-serverless-operator/pull/351